src/main/java/com/datalogics/pdf/samples/signature/SignDocument.java

```diff
@@ -74,6 +74,17 @@
     private static final Double MM_PER_INCH = 25.4; // millimeters per inch
     private static final Double DEFAULT_MM_PER_PIXEL = 0.35277778; // default millimeters per pixel
 
+    // Customize the label. You can provide your own string that describes who
+    // signed the document.  The "{0}" will be replaced by the name of the
+    // signer, which is taken from the UserInfo.name in the
```

Comment:
I think @datalogics-bhaugen was referring to line 79 ', which is taken from the UserInfo.name in the'